module/account/features/account.feature

```diff
@@ -792,14 +792,14 @@ Feature: Account module
       """
     Then the response status code should be 400
 
-  Scenario: Create user (long mail)
+  Scenario: Create user (long mail - 64 chars local part)
```

Comment:
A newer versions of `egulias/email-validator` used both within Symfony validators and Ergonode VO's allow only 64 chars in email part. This is valid according to RFC therefore adjusting to it.